velox/exec/tests/WindowFunctionRegistryTest.cpp

```diff
@@ -107,4 +117,87 @@ TEST_F(WindowFunctionRegistryTest, mixedCaseName) {
       "window_fUNC_alias", {BIGINT(), DOUBLE()}, BIGINT());
 }
 
+TEST_F(WindowFunctionRegistryTest, prefix) {
+  // Remove all functions and check for no entries.
+  exec::windowFunctions().clear();
+  EXPECT_EQ(0, exec::windowFunctions().size());
+
+  // Register without prefix and memorize function maps.
+  window::prestosql::registerAllWindowFunctions();
+  const auto windowFuncMapBase = exec::windowFunctions();
+
+  // Remove all functions and check for no entries.
+  exec::windowFunctions().clear();
+  EXPECT_EQ(0, exec::windowFunctions().size());
+
+  // Register with prefix and check all functions have the prefix.
+  const std::string prefix{"test.abc_schema."};
+  window::prestosql::registerAllWindowFunctions(prefix);
+  auto& windowFuncMap = exec::windowFunctions();
+  for (const auto& entry : windowFuncMap) {
+    EXPECT_EQ(prefix, entry.first.substr(0, prefix.size()));
+    EXPECT_EQ(1, windowFuncMapBase.count(entry.first.substr(prefix.size())));
+  }
+}
+
+TEST_F(WindowFunctionRegistryTest, missingFunctionSignature) {
```

Comment:
This is a better fit for WindowTest instead of WindowFunctionRegistryTest. Once you move it, then you leave WindowFunctionRegistryTest to derve from testing::Test instead of this change to use OperatorTestBase.

Reply:
Updated, thanks!